nvm.sh

```diff
@@ -1586,10 +1586,16 @@ nvm_install_binary() {
   local TMPDIR
   local VERSION_PATH
 
+  local PROGRESS_BAR
   local NODE_OR_IOJS
   if [ "${FLAVOR}" = 'node' ]; then
     NODE_OR_IOJS="${FLAVOR}"
   fi
+  if [ "${NVM_NO_PROGRESS}" = "1" ]; then
+    PROGRESS_BAR="-s"
```

Comment:
Please use:

```bash
PROGRESS_BAR="-sS"
```
`-s` by itself silences all output (including errors).  However, I believe a user would want to know why something failed so `-S` will still properly surface errors to them.  `-sS` will only output if there's an error.